data/mods/Xedra_Evolved/mutations/paraclesians/paraclesian_eocs.json

```
@@ -63,5 +63,14 @@
       }
     ],
     "false_effect": { "u_message": "You must be outdoors under the moonlight to turn cotton into gossamer." }
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_DESIRE_CONVERSATION",
+    "condition": {
+      "and": [ "u_is_npc", { "u_has_trait": "HOMULLUS_WISHES_TO_CONVERSE" }, { "not": { "u_has_trait": "FINAL_DISCUSSION" } } ]
```

```suggestion
      "and": [ "u_is_npc", { "u_has_trait": "HOMULLUS_WISHES_TO_CONVERSE" }, { "not": { "u_has_trait": "FINAL_DISCUSSION" } } ]
```
So the goal here was for the FINAL_DISCUSSION trait to pop on the NPC after their opinion is set to Wants to Talk once.  I can see it's not getting set here but it also looks like it is checking the PC for FINAL_DISCUSSION INSTEAD OF THE NPC or the NPC itself has to have HOMULLUS_WISHES_TO_CONVERSE?

---

Oh, I usually played as musclehead like Duke N****, so I didn't really know how to activate mutations( and their results ).
As a this mod, does it mean that the player will have a mutation of HOMULLUS_WISHES_TO_CONVERSE and intends to activate this to cause a conversation to be evoked on the target( NPC )?

I misunderstood what FINAL_DISCUSSION was given by an choice in conversation--dialogues-- ( like "Don't talk to me anymore." ), but it is given once HOMULLUS_WISHES_TO_CONVERSE is activated ( i.e., as the description in file, it causes him to talk to me only once )?
If my understanding of this time is correct, it is right to give FINAL_DISCUSSION to the NPC as you originally wanted it to be. Also, I think it is not necessary to check if the PC has FINAL_DISCUSSION.
And I think in this case ( if we want to activate HOMULLUS_WISHES_TO_CONVERSE to provoke a conversation with the NPC ), we would need select the target, or not?

---

Yeah it would need to select the target NPC, you are correct, I should have added that part. The thought process was that 1. There is a hostile NPC that won't talk.  2. The PC activates the ability, the NPC changes to "wanting to talk" and is given the hidden trait "Final_discussion"  3 The PC talks to the NPC, if the NPC goes hostile again the PC is now unable to force them to talk again.  Life has been crazy here but would you be able to get the json to do that?

---

That process is the ideal way to use this effect. 
I believe it is possible to implement this using only JSON.
And I feel that the example with the spell I added to the issues first topic can be adapted to it.

Will be add the refined spell examples to the JSON at a later( my virtual machine that I use for development stucks frequently with unknown keystroke delays, so I want to clean it up first ).

---

Code added to json.

When you get the `homullus_wishes_to_converse` trait, you automatically learn the `homullus_spell_wish_to_converse` spell.
The target of the spell gets the effect of `effect_wants_to_talk`.
Whoever gains the effect of `effect_wants_to_talk` will want to talk to you by EOC `EOC_WANTS_TO` and also gains the trait `FINAL_DISCUSSION`.
However, if the person has the trait `FINAL_DISCUSSION`, he/she will not want to talk to you initiatively.

---

It would be helpful if you could rewrite the name and description to appropriate ones.

---

"It would be helpful if you could rewrite the name and description to appropriate ones."

Oops what have I done that needs to be changed?  The trait name and description?  What would you suggest?

---

No, I have no problem with the name and description you wrote. 👍🏼 
I just wanted to ask you to fix it if you see any problem with the descriptions of the spell I added, from your perspective.

There 
https://github.com/CleverRaven/Cataclysm-DDA/blob/b597a2ba0fba805472e58ee27ed84b13192db021/data/mods/Xedra_Evolved/mutations/paraclesians/paraclesian_mutation_spells.json#L199

Also, the various parameters of the spell ( `base_casting_time`, `min_range`, etc.) are taken from nearby spells as reference and may not be appropriate. 
If there is a problem from your point ( practical point of view in the game ), I would appreciate it if you could please correct it. Unless there's any problem, it's fine.